specification/portal/resource-manager/Microsoft.Portal/preview/2020-09-01-preview/tenantConfiguration.json

```diff
@@ -0,0 +1,256 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-01-01-preview",
+    "title": "portal",
+    "description": "Manage tenant configuration.",
+    "x-ms-code-generation-settings": {
+      "name": "portalClient"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "paths": {
+    "/providers/Microsoft.Portal/tenantConfigurations": {
+      "get": {
+        "tags": [
+          "tenantConfiguration"
+        ],
+        "operationId": "TenantConfigurations_List",
+        "description": "Gets list of the tenant configurations.",
+        "parameters": [
+          {
+            "$ref": "portal.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Returns the list of tenant configurations.",
+            "schema": {
+              "$ref": "#/definitions/ConfigurationList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "portal.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get list of Tenant configurations": {
+            "$ref": "examples/TenantConfiguration/GetListOfTenantConfigurations.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Portal/tenantConfigurations/{configurationName}": {
+      "get": {
+        "tags": [
+          "tenantConfiguration"
+        ],
+        "operationId": "TenantConfigurations_Get",
+        "description": "Gets the tenant configuration.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ConfigurationNameParameter"
+          },
+          {
+            "$ref": "portal.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Returns the tenant configuration.",
+            "schema": {
+              "$ref": "#/definitions/Configuration"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "portal.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Tenant configuration": {
+            "$ref": "examples/TenantConfiguration/GetTenantConfiguration.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "tenantConfiguration"
+        ],
+        "operationId": "TenantConfigurations_Create",
+        "description": "Create the tenant configuration. If configuration already exists - update it. User has to be a Tenant Admin for this operation.",
```

Comment:
> User has to be a Tenant Admin for this operation." [](start = 101, length = 51)

What will happen to the already saved dashboards?

Reply:
Copying the swagger from 2019 version but as I understand the already saved dashboards would just show a message to the customer to fix the tiles that are out of compliance.

Reply:
More explanation on the tenant RP - again it has been merged in already earlier today. The tenant RP is for private dashboards which aren't ARM resources. Azure policy would be used for Shared dashboards compliance for which I am adding 2020-09-01-preview version.